src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -229,7 +230,33 @@ const formConfig = {
           }
         },
         // pageFour: {},
-        // pageFive: {},
+        privateMedicalRecordsIntro: {
+          title: '',
+          path: 'supporting-evidence/:index/private-medical-records-intro',
+          showPagePerItem: true,
+          arrayPath: 'disabilities',
+          depends: (formData, index) => _.get(`disabilities.${index}.view:privateMedicalRecords`, formData),
+          uiSchema: {
+            disabilities: {
+              items: {
+                'ui:title': disabilityNameTitle,
+                'ui:description': privateMedicalRecordsIntro,
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              disabilities: {
```

Comment:
~Should we rename one of the disabilities properties, either here or in the `privateRecordChoice` page?~
Should we create a component which like the photo cropper renders different views here rather than setting up objects without properties?